README.md

```diff
@@ -131,20 +131,28 @@ gulp.task('script', function() {
 
 ### Languages
 
-The Closure Compiler supports languages `ECMASCRIPT3`, `ECMASCRIPT5`, `ECMASCRIPT5_STRICT` for input and output, `ECMASCRIPT6`, `ECMASCRIPT6_STRICT` for input only, and `ECMASCRIPT6_TYPED` (experimental) for both.
+The Closure Compiler supports the following languages:
+- `ECMASCRIPT3`, `ECMASCRIPT5` and `ECMASCRIPT5_STRICT` for input and output;
+- `ECMASCRIPT6` and `ECMASCRIPT6_STRICT` for input only;
+-  `ECMASCRIPT6_TYPED` (experimental) for both.
```

Comment:
We're actually moving to "ECMASCRIPT_2015" instead of "6" but this is fine for right now.